spec/models/reception_spec.rb

```diff
@@ -36,4 +36,118 @@
 
     it { is_expected.not_to be_valid }
   end
+
+  context 'without plate_attributes or tubes_attributes' do
+    let(:attributes) { { plates_attributes: [], tubes_attributes: [] } }
+
+    it { is_expected.not_to be_valid }
+  end
+
+  describe '#construct_resources!' do
```

Comment:
We already test construct_resources extensively in the resource factory so this can just be a basic test. We just want to test the association works so something like `expect reception.construct_resources! to change reception.requests by x`

Reply:
Fair enough.  I thought all the other stuff was necessary for the resource test to work correctly, but I've reduced it right down now.